oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/Column.java

```diff
@@ -102,6 +101,28 @@
      */
     AnalyzerType analyzer() default AnalyzerType.OAP_ANALYZER;
 
+    /**
+     * Sharding key is used to group time series data per metric of one entity.
+     * For example,
+     * ServiceA's traffic gauge, service call per minute, includes following timestamp values, then it should be shard
+     * by service ID
+     * [ServiceA(encoded ID): 01-28 18:30 values-1, 01-28 18:31 values-2, 01-28 18:32 values-3, 01-28 18:32 values-4]
+     *
+     * BanyanDB is the 1st storage implementation supporting this. It would make continuous time series metrics stored
+     * closely and compressed better.
+     *
+     * 1. One entity at most has one sharding key
```

Comment:
No. We support sharding with multiple keys now.

Reply:
@hanahmily What do you prefer? I think you mentioned not to use multiple keys.